package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "homepage": "https://github.com/openmrs/openmrs-form-engine-lib#readme",
   "scripts": {
-    "lint": "TIMING=1 eslint src --ext .ts,.tsx --fix",
```

Comment:
We no longer need this `TIMING` environment variable as all output to `stdout` gets cached by turbo.